src/zenml/stack_stores/sql_stack_store.py

```diff
@@ -0,0 +1,313 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+# type: ignore
+# ^ this is needed here in order to keep SQLModel from blowing up mypy.
```

Comment:
What errors do you get from mypy when uncommenting this line? We might be able to add some `sqlmodel.*`ignore in `pyproject.toml` to ignore some errors for this specific package or disable type checking for it entirely

Reply:
This is a weird one. it doesn't just error in the sense that I could ignore that, the entire mypy run fails before checking a single file in the project.

```
±  mypy src
Traceback (most recent call last):
  File "/Users/jwb/Developer/zenml/venvs/zenintel3/bin/mypy", line 8, in <module>
    sys.exit(console_entry())
  File "/Users/jwb/Developer/zenml/venvs/zenintel3/lib/python3.8/site-packages/mypy/__main__.py", line 12, in console_entry
    main(None, sys.stdout, sys.stderr)
  File "mypy/main.py", line 96, in main
  File "mypy/main.py", line 173, in run_build
  File "mypy/build.py", line 180, in build
  File "mypy/build.py", line 256, in _build
  File "mypy/build.py", line 2717, in dispatch
  File "mypy/build.py", line 3048, in process_graph
  File "mypy/build.py", line 3165, in process_stale_scc
  File "mypy/build.py", line 2308, in write_cache
  File "mypy/build.py", line 1490, in write_cache
  File "mypy/nodes.py", line 334, in serialize
  File "mypy/nodes.py", line 3330, in serialize
  File "mypy/nodes.py", line 3267, in serialize
  File "mypy/nodes.py", line 941, in serialize
  File "mypy/types.py", line 1386, in serialize
  File "mypy/types.py", line 541, in serialize
  File "mypy/types.py", line 984, in serialize
  File "mypy/nodes.py", line 2642, in fullname
AttributeError: attribute '_fullname' of 'TypeInfo' undefined
```

Any ideas?

Reply:
No idea, really weird that it's an actual mypy internal crash. I'll see if the same happens for me or if I can find a solution to it